Packs/Cyberint/ReleaseNotes/1_1_3.md

```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Cyberint
+
+- Updated the **fetch-alert-list** command.
+- Added support for mirroring.
+- Updated the Docker image to: *demisto/python3:3.10.13.78960*.
```

Comment:
looks like there are also changes in cyberint-alerts-status-update

Reply:
Updated